analyzer/library/bundler/bundler.go

```diff
@@ -37,7 +36,11 @@ func (a bundlerLibraryAnalyzer) Analyze(fileMap extractor.FileMap) (map[types.Fi
 		if err != nil {
 			return nil, xerrors.Errorf("error with %s: %w", filename, err)
 		}
-		libMap[types.FilePath(filename)] = libs
+		for _, lib := range libs {
+			libMap[types.FilePath(filename)] = append(libMap[types.FilePath(filename)], types.LibraryInfo{
+				Library: lib,
+			})
+		}
```

Comment:
I feel we should fix it here.
https://github.com/aquasecurity/fanal/blob/e6c70ea926f2d03f8f40b8d85023eef6dd1ef2c8/analyzer/analyzer.go#L319

Reply:
Good point.